kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build-model-image.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# 
+#  Summary:
+#
+#    This script builds a model image using the WebLogic Image Tool. The
+#    tool pulls a base image if there isn't already a local base image.
+#    This script, by default, builds the model image with model files from
+#    './stage-model-image.sh' plus tooling downloaded by './stage-tooling.sh.'.
+#
+#  Assumptions:
+#
+#    - The WebLogic Image Tool zip is 'WORKDIR/weblogic-image-tool.zip' and
```

Comment:
Image tool has the concept of a cache directory, which you then reference.  It feels like we are just adding a bunch of machinery on top of the image tool.

Reply:
What machinery is added?  This particular script expects some staged files.   The image tool requires them.  The staged files are setup with two small scripts - stage-tooling and stage-model-image.  Are you saying that's too much?

Reply:
Note that I'm under the impression image tool 'cache' concept is mainly intended for caching installers, and that's how it's being used in the sample.  This is the first I've used of it being used for caching model files.